### DISCUSSION TITLE
maximum_momentum

### DISCUSSION THREAD

**Discussion by MSAfify** [danielver02/ALPS#165] [Q&A] [answered]
Hi Prof. Daniel,
Please let me know what the difference is between maxP, maxPperpS, and maxPparS, which appear in the test_ICW_dist.in file in the distribution folder.

* In the ALPS paper, what is the meaning of M_I and M_P on page 10? Which symbols refer to such variables in the input file?
* Also, for P_max||p = 6m_p v_A, the normalized parallel momentum should be P_max||p=6, right?
Thanks!
Mahmoud

**Comment by danielver02** [comment, marked as answer]
Hi Mahmoud,

With maxPperpS and maxPparS, you can set different maximum momenta for each species. maxP defines a maximum for the first species, and the code then tries to find the best scaling if you set autoscaleS to TRUE. If you want to force a defined maximum momentum via maxPperpS and maxPparS, then autoscale must be set to FALSE.

M_I and M_P are the variables positions_principal and n_resonance_interval. Please read the code documentation carefully, as these quantities are explained and defined there. Here is the link: https://danielver02.github.io/ALPS/page/input.html 

Yes, your question about the momentum is correct. All momentum units are given in units of m_p v_A (i.e., the Alfvén momentum of the reference species). 

Best wishes,
-Daniel

**Reply by MSAfify** [reply]
Hi Prof. Daniel,
Great thanks!
Mahmoud
